cmd/exo/cmd/root.go

```diff
@@ -104,15 +105,56 @@ func initConfig() {
 		break
 	}
 
-	if configFilePath == "" {
-		path, err := generateConfigFile(false)
-		if err != nil {
-			log.Fatal(err)
-		}
-		configFilePath = path
+	if cfgFilePath != "" {
+		configFilePath = cfgFilePath
+		return
+	}
+
+	if getCmdPosition("config") == 1 {
+		ignoreBuilClient = true
```

Comment:
typo?

Reply:
`ignoreClientBuild` instead of `ignoreBuilClient` and missing `d` to build ! thanks 👍